data/monaco_amenities.txt

```diff
@@ -0,0 +1,308 @@
+Chemin des Pêcheurs
```

Comment:
Similar to the `.osm.pbf`: I don't think we should store this data in Git.

Maybe create a `.gitignore` and put the pbf and this cache on it?

Reply:
Cool, sounds good 👍